.github/workflows/acceptance-py3.6.yml

```diff
@@ -18,6 +18,7 @@ jobs:
 
     env:
       MIGRATIONS_TEST_MIGRATE: 1
+      TEST_GROUP_STRATEGY: individual
```

Comment:
This doesn't actually mean anything right now, only that it's not `file`. I'd rename to `roundrobin`, this confused me for a second.